apps/web/src/components/Main/RoomCardList.tsx

```
@@ -48,12 +49,17 @@ function RoomCardList({ rooms, onSpectate, joiningRoomId }: Props) {
+                  <span aria-hidden="true">●</span> LIVE
+                </span>
                 <span className="text-blue-03">{formatElapsed(room.createdAt)}</span>
               </div>
-              <div className="inline-flex items-center gap-1.5 text-blue-03 text-sm font-medium">
-                <Eye className="h-4 w-4 text-blue-03" strokeWidth={1.5} />
-                {room.currentSpectators?.length ?? 0}
+              <div
+                className="inline-flex items-center gap-1.5 text-blue-03 text-sm font-medium"
+                aria-label={`관전자 ${room.currentSpectators?.length ?? 0}명`}
+              >
+                <Eye className="h-4 w-4 text-blue-03" strokeWidth={1.5} aria-hidden="true" />
```

Biome (2.3.13)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)